modules/machine-health-checks-about.adoc

```diff
@@ -29,8 +29,55 @@ Consider the timeouts carefully, accounting for workloads and requirements.
 
 To stop the check, remove the resource.
 
-For example, you should stop the check during the upgrade process because the nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
-However, a `MachineHealthCheck` resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
+
+[id="machine-health-checks-pause_{context}"]
```

Comment:
This is procedural and should be in a separate procedure module, not added to a concept. Please move this to a separate module and reformat it to match the procedure template.

Reply:
Accepted and updated